packages/static-wado-scp/lib/program/index.js

```diff
@@ -2,8 +2,16 @@ const staticWadoUtil = require("@ohif/static-wado-util");
 const { DcmjsDimseScp, Server } = require("..");
 const packageJson = require("../../package.json");
 
+const aeConfig = staticWadoUtil.aeConfig;
+
 function main() {
-  const port = this.dicomWebScpConfig.scpPort || 11112;
+  const { queryAe } = this.dicomWebScpConfig;
+  if (!queryAe) throw new Error("queryAe not specified");
+  const queryAeData = aeConfig[queryAe];
+
+  if (!queryAeData) throw new Error(`No data for aeConfig.${queryAe} is configured in ${Object.keys(aeConfig)}`);
+
+  const port = queryAeData.port || 11112;
```

Comment:
The port number here is the listen port number, whereas the queryAe port is the port number on the queried remote system.